Homework_5/Task_1.py

```diff
@@ -0,0 +1,9 @@
+str1 = input("Enter the string")
+
+str1 = str1.replace(' ','').lower()
+palindrome = str1 == ''.join(reversed(str1))
+
+if palindrome:
+    print("string is polendrome")
+else:
+    print("string is not polendrome")
```

Comment:
Ok. But it can be simpler
actual_str = input("Enter the string: ")
print(f'Is string {actual_str} palindrome? {actual_str == actual_str[::-1]}')

Reply:
Done!